case/initcond.F90

```diff
@@ -94,9 +95,13 @@ program InitCond
         print*, 'Xc', iz, xc
 
         rbc => rbcs(iz)
-        rbc%celltype = 1
-        call Rbc_Create(rbc, nlat0, dealias)
-        call Rbc_MakeBiConcave(rbc, radEqv, xc)
+        if (iz .eq. 1) then 
+          rbc%celltype = 1
+          call Rbc_Create(rbc, nlat0, dealias)
+          call RBC_MakeBiconcave(rbc, radEqv, xc)
+        else 
+          call ReadRBCPlain('Input/SickleCell.dat', rbc, xc)
```

Comment:
so plain is sickle? also, what it seems like the input data depends on the values of `nlat` and `nlon`, or no? if so, what does one do if they want a sickle cell with different nlat and nlon?

Reply:
Yes, in this case, we have a file of a Sickle Cell (previously exported) which is imported using the ReadRBCPlain subroutine. I will fix naming and explain better via comments in next revision.

The imported cell will be the same exact shape as the cell which was previously exported to the defined file. So, if we want a Sickle Cell with different nlat/nlon, we would have to recreate the sickle cell with those different measurements and then re-export it.